include/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+#----------------------------------------------------------------------
+# Public includes
+
+# Interface target to allow headers to be "linked" into other targets.
+add_library(headers INTERFACE)
+
+# Installation location for install phase.
+install(
+    DIRECTORY
+    ${CMAKE_CURRENT_LIST_DIR}/openassetio
+    DESTINATION
+    ${CMAKE_INSTALL_INCLUDEDIR}/
+)
+
+target_include_directories(headers
```

Comment:
This I would move to the source compilation target.

Reply:
Moved the public header config alongside the target config in 99d122c